File: trunk/application/models/Friend.php
<?php
//include_once "system/libraries/Model.php";
class Friend  {

    private $gwt_class = "test.client.Friend";
    private $name = '';
    private $phoneNumber = '';
    private $birthday = '';

    function Friend() {
    // Call the Model constructor
       // parent::Model();
    }

    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getBirthday() {
        return $this->birthday;
    }

    public function setBirthday($birthday) {
        $this->birthday = $birthday;
    }

    public function getPhoneNumber() {
        return $this->phoneNumber;
    }

    public function setPhoneNumber($phoneNumber) {
        $this->phoneNumber = $phoneNumber;
    }

    public function getGwt_class() {
        return $this->gwt_class;
    }

}
?>

File: k2-client/src/test/client/Friend.java
package test.client;


public class Friend implements BaseModel{
	
	public static final String $php_class = "Friend";
    private String name = "";
    private String phoneNumber = "";
    private String birthday = "";
    
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getPhoneNumber() {
		return phoneNumber;
	}
	public void setPhoneNumber(String phoneNumber) {
		this.phoneNumber = phoneNumber;
	}
	public String getBirthday() {
		return birthday;
	}
	public void setBirthday(String birthday) {
		this.birthday = birthday;
	}

}
